tutorials/02a_RunningREMINDLocally.md

```diff
@@ -1,10 +1,146 @@
 # Running REMIND locally
-Mika Pflüger (mika.pflueger@pik-potsdam.de)
+Mika Pflüger (mika.pflueger@pik-potsdam.de), Tonn Rüter (tonn.rueter@pik-potsdam.de)
 
-*Note:* This tutorial is a work in progress, not all steps needed are described, yet.
+*Note:* This approach of running REMIND is a work in progress, however successful REMIND runs on local machines have been conducted using this tutorial.
+
+## System Requirements
+
+- Installed python version must be `3.10.X` or older. To check your python version run `python --version` in a terminal of your choice
+- GAMS must be installed and the path to GAMS applications needs to be in your PATH environment variable
+- R version must be at least `4.2.X`. To check your R version run `R --version` in a terminal of your choice
```

Comment:
Where does that come from?
We should update [the `DESCRIPTION` file](https://github.com/remindmodel/remind/blob/9f6a2cb01f19b783dd8823ae0e6616c0c658962d/DESCRIPTION#L12) then.

Reply:
Points well taken.

The thing is: this is a replication and as such a source of inconsistency.  An undiscerning user might look at this tutorial, and never the `DESCRIPTION` file, which is supposed to contain the "real" dependencies.
I would argue for telling people to run `make ensure-reqs` (or `Rscript -e 'source("scripts/start/ensureRequirementsInstalled.R"); ensureRequirementsInstalled(rerunPrompt="make ensure-reqs")'` if they do not have that) and note here that things are easier with `R 4.2` on Windows (on Linux, everything is installed from source anyway).

I wonder: who is the target audience for this tutorial, i.e. who wants to run REMIND locally (and on Windows)? @LaviniaBaumstark 

If that is "REMIND team members working on their laptop instead of the cluster", we could reasonably expect them to get a work environment that includes `make` (and support them in doing so), since the tests they are expected to perform in the course of their work depend on it.

If that is "REMIND team members" plus "outside interested parties", we might have a "default" description for a full work environment and a striped-down version for those who just want to dabble.

-----

> Updating the DESCRIPTION file to R (> 4.1.3) would therefore break the REMIND cluster deployment

Nothing that could not be overcome with one edit by RSE …
```
$ module --terse available 2>&1 | grep 'R/4.2'
R/4.2.0/gcc-mkl
```
;)

Reply:
Maybe just create a separate section with additional remarks for Windows users. There, you can mention that one should better use an R version which still offers binaries, as you probably don't want to compile the R libs on your Windows machine.

Then, you can simply refer to the requirements in Chapter 01 Getting REMIND.

Reply:
> I wonder: who is the target audience for this tutorial, i.e. who wants to run REMIND locally (and on Windows)? @LaviniaBaumstark

We do have a request for this.
I agree, Windows should not be the recommended OS to use. This is why I would move all the Windows-specific recommendations to a separate section at the bottom, starting with sth along the lines of "We do not recommend it, but if you must.."

Reply:
> you can mention that one should better use an R version which still offers binaries, as you probably don't want to compile the R libs on your Windows machine.

Amazing that we assume people wanting to run REMIND cannot be troubled to wait half an hour to set up the required environment.